src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/ZonedDateTimeSerializer.java

```diff
@@ -8,6 +9,8 @@
 
     protected ZonedDateTimeSerializer() {
         super(ZonedDateTime.class, dt -> dt.toInstant().toEpochMilli(),
-                ZonedDateTime::toEpochSecond, ZonedDateTime::getNano);
+                ZonedDateTime::toEpochSecond, ZonedDateTime::getNano,
+                // ISO_ZONED_DATE_TIME is not the ISO format, it is an extension of it
+                dt -> DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(dt));
```

Comment:
Yes, this PR only contained the ISO compatibility changes and introduced loss of time zone information. The idea of the `SerializationFeature` came up later.